src/theme.js

```diff
@@ -12,6 +22,16 @@ export const breakpoints = [
 
 export const mediaQueries = breakpoints.map(createMediaQuery)
 
+const aliases = [
+  'sm',
+  'md',
+  'lg',
+  'xl'
```

Comment:
I like these aliases, but I know some prefer to have things spelled out more explicitly (e.g. `xsmall`, `medium`), do you think there's a general preference one way or another among your team members?

Reply:
My vote goes to having these aliases spelled out. (i.e. the longest one here would be `extraLarge`; still only 10 letters) The benefit is to have clearly expressed terminology to avoid ambiguity, while the effort is still at most 10 letters. 😄

Reply:
@BeniCheni We could have both 😂